8.0/Dockerfile

```diff
@@ -0,0 +1,95 @@
+FROM debian:buster
+
+RUN \
+  apt-get update && \
+  apt-get install -y \
+  curl \
+  wget \
+  apt-transport-https \
+  lsb-release \
+  ca-certificates \
+  gnupg2 \
+  git
+
+RUN wget -O- https://packages.sury.org/php/apt.gpg | apt-key add - && \
+    echo "deb https://packages.sury.org/php/ $(lsb_release -sc) main" > /etc/apt/sources.list.d/php.list
+
+RUN \
+  apt-get update && \
+  apt-get install -y \
+  vim \
+  locales \
+  php8.0-fpm \
+  php8.0-mysql \
+  php8.0-gd \
+  php8.0-imagick \
+  php8.0-dev \
+  php8.0-curl \
+  php8.0-opcache \
+  php8.0-cli \
+  php8.0-sqlite \
+  php8.0-intl \
+  php8.0-tidy \
+  php8.0-imap \
+  php8.0-json \
+  php8.0-pspell \
+  php8.0-common \
+  php8.0-sybase \
+  php8.0-sqlite3 \
+  php8.0-bz2 \
+  php8.0-common \
+  php8.0-apcu-bc \
+  php8.0-memcached \
+  php8.0-redis \
+  php8.0-xml \
+  php8.0-shmop \
+  php8.0-mbstring \
+  php8.0-zip \
+  php8.0-bcmath \
+  php8.0-soap
+
+RUN curl -sS https://getcomposer.org/installer | php -- --install-dir=/usr/local/bin --filename=composer && \
+    composer global require hirak/prestissimo && \
+    wget http://robo.li/robo.phar && \
```

Comment:
Do we still need to include robo in the main image? I think this should be a project level requirement.